src/pages/guides/sellers/marketing-review-guidelines.md

```diff
@@ -55,11 +55,11 @@ For guides and examples of long descriptions, see [product descriptions](product
 
 ## User guide, reference manual, or installation guide
 
-Extension documentation should fulfill the following guidelines:
+Product documentation should fulfill the following guidelines:
 
-*  Must cover all features of the extension.
+*  Must cover all features of the product.
 *  Must not direct users to make purchases on sites other than Marketplace.
-*  If an extension title is added into the documentation, it must not include _Magento_. Use _For Magento_ after the title if needed.
+*  If an product title is added into the documentation, it must not include _Magento_. Use _For Magento_ after the title if needed.
 *  Use of the Magento logo is prohibited. To learn more, see our [branding guidelines](branding.md).
    *  Partners can use their Magento partner badge.
 *  At least one piece of uploaded documentation must be in English.
```

Comment:
On the content section, please update the sentence "The content that accompanies your extension listing is derived from the marketing information you entered for the extension or theme, and information in your company profile. To learn more, see extension content" to something like 

"The content that accompanies your listing is derived from the marketing information you entered for the extension, app or theme, and information in your company profile. To learn more, see listings content"